test.js

```diff
@@ -1,57 +1,109 @@
 const { describe, it } = require("node:test");
 const assert = require("node:assert");
 
-const index = require("./index.js");
-
-describe("Test suite: extractGitHubRepoPath", () => {
```

Comment:
Would be nice to have different `describe`s for the two different methods tested: `extractGitHubRepoData` and `buildGitHubUrl`. So in that way we can structure our test suites.